### PR TITLE
Less messy login page

### LINKED ISSUE #200: Tidy up the login page
Lots of text has been added to this page recently. Make it less overwhelming.

### PR DESCRIPTION
**Pull request by zarino**
Fixes #200. Text on the login page is now slightly smaller, the login buttons slightly higher, and the line lengths more comfortable on wide screens.

It's still a bit wordy, but unless @MyfanwyNixon fancies shortening the copy, this is ready to go.

screen shot 2015-07-29 at 10 42 38

### CODE REVIEW

**Comment by tmtmtmtm** [comment]
Looks good to me. I think there's a slightly overlap between the two sections of text, as they were each written independently, so I'm definitely open to suggestions for @MyfanwyNixon to harmonise and streamline the whole thing. But that can go in as a separate change if so…  :shipit:

**Comment by MyfanwyNixon** [comment]
I think the quickest solution would be to drop "That’s pretty cool, actually, because it will be available for teachers, researchers, journalists, analysts and people who want to do good things for the world."
